wp-admin/css/widgets/media-widgets.css

```diff
@@ -47,7 +47,8 @@
 .media-frame.media-widget .image-details .embed-media-settings .setting.align,
 .media-frame.media-widget .attachment-display-settings .setting.align,
 .media-frame.media-widget .embed-media-settings .setting.align,
-.media-frame.media-widget .embed-link-settings .setting.link-text {
+.media-frame.media-widget .embed-link-settings .setting.link-text,
+.media-frame.media-widget .replace-attachment {
```

Comment:
There is no underlying state that controls whether or not the Replace button is shown. So we have to rely on CSS instead.